xgboost_ames_housing/housing.py

```diff
@@ -61,7 +62,7 @@ def train_model(train_X,
 def eval_model(model, test_X, test_y):
   """Evaluate the model performance."""
   predictions = model.predict(test_X)
-  print("MAE on test: {:.2f}".format(mean_absolute_error(predictions, test_y)))
+  print("mean_absolute_error={:.2f}".format(mean_absolute_error(predictions, test_y)))
```

Comment:
Should we use the logging module rather than print?

Reply:
Fixed